src/vinyldns/client.py

```diff
@@ -622,12 +631,39 @@ def list_batch_change_summaries(self, start_from=None, max_items=None, **kwargs)
             args.append(u'startFrom={0}'.format(start_from))
         if max_items is not None:
             args.append(u'maxItems={0}'.format(max_items))
+        if ignore_access:
+            args.append(u'ignoreAccess={0}'.format(ignore_access))
+        if approval_status:
+            args.append(u'approvalStatus={0}'.format(approval_status))
 
         url = urljoin(self.index_url, u'/zones/batchrecordchanges') + u'?' + u'&'.join(args)
 
         response, data = self.__make_request(url, u'GET', self.headers, **kwargs)
         return ListBatchChangeSummaries.from_dict(data)
 
+    def approve_batch_change(self, batch_change_id, approval=None, **kwargs):
+        """
+        Approve a batch change
+
+        :return: the content of the response
+        """
+        url = urljoin(self.index_url, '/zones/batchrecordchanges/{0}/approve'.format(batch_change_id),
+                      to_json_string(approval))
+        response, data = self.__make_request(url, u'POST', self.headers, to_json_string(approval), **kwargs)
```

Comment:
also of note, not sure how error handling should be working here, but it's definitely throwing. following example:
```
>>> client.approve_batch_change("asdasd")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/rstar201/Desktop/Vinyl/python/vinyldns-python/src/vinyldns/client.py", line 654, in approve_batch_change
    return BatchChange.from_dict(data)
  File "/Users/rstar201/Desktop/Vinyl/python/vinyldns-python/src/vinyldns/batch_change.py", line 211, in from_dict
    user_id=d['userId'],
TypeError: 'NoneType' object is not subscriptable
```

Reply:
maybe another @pauljamescleary question, what kind of error handling do we have in this client?

Reply:
I updated the 404 error to raise instead of return and that seemed to clear this up and return the expected error, that the Batch Change with id "asdasd" could not be found. @pauljamescleary let me know if i shouldn't be raising on the 404 and what you'd suggest instead.